compiler/test/test_end_to_end.re

```diff
@@ -865,7 +865,7 @@ let box_tests = [
     "let b = box(4);\n            {\n              b := unbox(b) - 1;\n              unbox(b)\n            }",
     "3",
   ),
-  t("test_set_extra1", "box(1) := 2", "2"),
+  t("test_set_extra1", "box(1) := 2", "void"),
```

Comment:
The tests below are actually doubled! There's a test for the assignment itself (which should return void) and the following test where it unboxes the value and checks that the value is correct. It's important to check that the return value is correct since even though typechecking is correct, we could accidentally generate the wrong code.

For this particular test, it seemed more like we were testing the return value of the statement, so I changed the expectation. I can also double this test to check the unboxed value as well, though.

Reply:
Makes sense. Our test names leave a lot to be desired.